pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -1026,3 +997,28 @@ def _set_distrib_type_if_training_type_plugin_passed(self):
             return
         if self._training_type_plugin is not None:
             self._distrib_type = getattr(self._training_type_plugin, "distributed_backend", None)
+
+    @property
+    def _is_slurm_managing_tasks(self) -> bool:
+        """Returns whether we let SLURM manage the processes or not.
+
+        Returns ``True`` if and only if these conditions match:
+
+            - A SLURM cluster is detected
+            - A distributed plugin is being used
+            - The process is not launching in interactive mode
+            - The number of tasks in SLURM matches the requested number of devices and nodes in the Trainer
+        """
+        if not self.use_ddp and not self.use_ddp2:
+            return False
+
+        if not SLURMEnvironment.is_using_slurm():
+            return False
+
+        if os.environ.get("SLURM_JOB_NAME") == "bash":
+            # in interactive mode we don't manage tasks
+            return False
+
+        total_requested_devices = len(self.parallel_devices) * self.num_nodes
+        num_slurm_tasks = int(os.environ["SLURM_NTASKS"], 0)
+        return num_slurm_tasks == total_requested_devices
```

Comment:
It would be great to print a info message to the Users that Slurm is being used.